clio-client/src/main/scala/org/broadinstitute/clio/client/commands/ClioCommand.scala

```diff
@@ -1,10 +1,17 @@
 package org.broadinstitute.clio.client.commands
 
 import org.broadinstitute.clio.transfer.model._
-
 import caseapp.{CommandName, Recurse}
 import caseapp.core.help.CommandsHelp
 import caseapp.core.commandparser.CommandParser
+import org.broadinstitute.clio.transfer.model.gvcf.{
+  TransferGvcfV1Key,
+  TransferGvcfV1QueryInput
+}
+import org.broadinstitute.clio.transfer.model.wgsubam.{
+  TransferWgsUbamV1Key,
+  TransferWgsUbamV1QueryInput
+}
```

Comment:
Here and elsewhere: the `model` package was getting pretty packed, so I divided it up. The fallout is scattered throughout the PR.